Components/WebService/Rest/RestConfigurableProtocol.php

```diff
@@ -72,6 +75,7 @@ public function configureOptionsResolver(OptionsResolver $resolver)
         $resolver->setAllowedTypes(self::OPTION_AUTH, ['string', 'null']);
         $resolver->setAllowedTypes(self::OPTION_ENCODING, ['string']);
         $resolver->setAllowedValues(self::OPTION_ENCODING, [self::ENCODING_JSON, self::ENCODING_XML]);
+        $resolver->setAllowedTypes(self::OPTION_RESPONSE_FALLBACK_ON_ERROR, ['boolean']);
```

Comment:
The type should be 'bool' instead of boolean. Symfony itself has defined in this way. Checking the internal code of the resolver, if I didn't miss anything, when checking the type does this
```
private static function isValueValidType($type, $value)
{
    return (\function_exists($isFunction = 'is_'.$type) && $isFunction($value)) || $value instanceof $type;
}
```

Reply:
`boolean` it's an alias of `bool`: https://github.com/symfony/options-resolver/blob/3.4/OptionsResolver.php#L89

Aliases are available in the latest version of Symfony as well.

Reply:
I see they also have this for symfony 2.8 https://github.com/symfony/options-resolver/blob/2.8/OptionsResolver.php , so we are safe